apps/storybook/package.json

```diff
@@ -34,7 +34,7 @@
     "vite": "^4.1.5"
   },
   "scripts": {
-    "build": "dotenv -v NODE_OPTIONS=\"--max-old-space-size=4096\" sb build",
+    "build": "tsc -p tsconfig.build.json && dotenv -v NODE_OPTIONS=\"--max-old-space-size=4096\" sb build",
```

Comment:
this is failing right now
<img width="697" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/6601bdee-4a27-4ec8-bcdf-d367e6dc7b24">

Reply:
actually this was an issue on my end, nvm